src/components/structures/auth/Registration.js

```diff
@@ -555,6 +555,7 @@ module.exports = React.createClass({
                 flows={this.state.flows}
                 serverConfig={this.props.serverConfig}
                 canSubmit={!this.state.serverErrorIsFatal}
+                idServerUrl={this.props.serverConfig.isUrl}
```

Comment:
do we need to pass the URL specifically through when we already pass the whole config?

Reply:
oh, good point